FWCore/ParameterSet/python/Config.py

```diff
@@ -1298,7 +1298,7 @@ def _insertPaths(self, processPSet, nodeVisitor):
               l.sort()
               decoratedList.extend(l)
               decoratedList.append("@")
-            iPath.insertInto(processPSet, triggername, decoratedList)
+            iPath.insertInto(processPSet, triggername, decoratedList[:])
```

Comment:
`decoratedList` is copied because otherwise the same `list` object would be shared among the Paths, and the contents would be that of the last Path.